projects/packages/jetpack-mu-wpcom/src/features/wpcom-dashboard-widgets/wpcom-dashboard-widgets.php

```diff
@@ -13,7 +15,36 @@ function load_wpcom_dashboard_widgets() {
 		return;
 	}
 
-	enqueue_wpcom_dashboard_widgets();
+	$layout_response = Client::wpcom_json_api_request_as_blog(
```

Comment:
Oh, you need to `request_as_user` here, then you won't need to "fix" authentication on the server.

Reply:
Huh! That works! 😮 Thank you!

Reply:
`wpcom_json_api_request_as_user` does not seem to work on dot com, so I've opted for using `wpcom_json_api_request_as_blog` in that case for now. Only that method seems to support internal requests. I wonder if there's anything preventing us from adding it to `wpcom_json_api_request_as_user`.

Reply:
> `wpcom_json_api_request_as_user` does not seem to work on dot com

What does it do instead of working? And what does "on dot com" mean? On a Simple site? Or on `public-api`? A Simple site should never call `request_as_user` or `request_as_blog` in the first place: that means sending a REST request basically to itself.

Reply:
It should make an internal request in that case, see https://github.com/Automattic/jetpack/pull/41864

Reply:
Oh, this `wpcom-api-direct` thing is very interesting. Maybe that's what we need to make Gutenberg preloading work 🙂